console/web3ext/web3ext.go

```diff
@@ -291,6 +291,11 @@ web3._extend({
 			name: 'stopStateMigration',
 			call: 'admin_stopStateMigration',
 		}),
+		new web3._extend.Method({
+			name: 'saveTrieNodeToDisk',
```

Comment:
How about this? :)
```suggestion
			name: 'saveTrieNodeCacheToDisk',
```

Reply:
@KimKyungup That would be better. I will change this!